scenes/main.tscn

```diff
@@ -118,7 +122,17 @@ position = Vector2(288, -84)
 texture = ExtResource("13_jbj1t")
 
 [node name="Sheep" parent="OnTheGround/MusicPuzzle" instance=ExtResource("14_a8y0u")]
-position = Vector2(304, 228)
+position = Vector2(672, 252)
+
+[node name="Teleporter" type="Area2D" parent="."]
+position = Vector2(2117, 1023)
+script = ExtResource("15_jkv2x")
+scene_to_go_to = "uid://db6vhi7s2f37c"
+metadata/_custom_type_script = "uid://hqdquinbimce"
```

Comment:
I imagine this would be replaced by a collectible which, when collected, triggers some victory animation and then a scene transition.